src/compass/utils/lut.py

```diff
@@ -97,9 +100,11 @@ def compute_geocoding_correction_luts(burst, dem_path,
         S1-A/B burst object
     dem_path: str
         Path to the DEM required for azimuth FM rate mismatch.
-    xstep: int
+    tec_path: str
+        Path to the TEC file for ionosphere correction
+    rg_step: int
         LUT spacing along x/slant range in meters
-    ystep: int
+    az_ystep: float
```

Comment:
This does not reflect the name of the function variable

Reply:
```suggestion
    az_step: float
```

Reply:
Thanks for catching this. That was a typo.